src/cpp/session/modules/SessionTutorial.cpp

```diff
@@ -249,8 +225,11 @@ void handleTutorialHomeRequest(const http::Request& request,
             << htmlEscape(tutorial.title)
             << "</span>";
          
-         ss << "<span class=\"rstudio-tutorials-name\">"
-            << htmlFormatTutorialName(pkgName, tutorial)
+         ss << "<span class=\"rstudio-tutorials-run-container\">"
+            << "<button class=\"rstudio-tutorials-run-button\" onclick=\"window.parent.tutorialRun('" << htmlEscape(tutorial.name) << "', '" << htmlEscape(pkgName) << "')\">"
+            << "<span class=\"rstudio-tutorials-run-button-label\">Start Tutorial</span>"
```

Comment:
The purpose of the button is obvious to a sighted user, but to a screen-reader user the buttons look like a list of identical buttons with no context. Example below from VoiceOver.

To fix this, add an `aria-label="foo"` to each button where "foo" is something like "Start Data Basics Tutorial". This will be read by screen readers, but visual users will just see the existing label.

![IMG_4076](https://user-images.githubusercontent.com/10569626/74883082-0d48ed80-5325-11ea-93f6-64d2cd0bd4d3.JPG)